server/app/views/admin/questions/QuestionsListView.java

```diff
@@ -238,19 +241,22 @@ private Pair<TdTag, Optional<Modal>> renderReferencingPrograms(
             .with(span(" programs"))
             .withClass(Styles.FONT_SEMIBOLD);
 
-    ContainerTag referencingProgramsCountContainer = referencingProgramsCount;
-    if (maybeReferencingProgramsModal.isPresent()) {
-      referencingProgramsCountContainer =
-          a().withId(maybeReferencingProgramsModal.get().getTriggerButtonId())
-              .withClasses(Styles.DECORATION_SOLID, Styles.CURSOR_POINTER)
-              .with(referencingProgramsCount);
-    }
-
     TdTag tag =
-        td().with(p().with(span("Used across "), referencingProgramsCountContainer))
+        td().with(p().with(span("Used across "), referencingProgramsCount))
```

Comment:
I'd confirm with @rahaghas about having a "View List" link versus linkifying the count (as before).

Reply:
Mocks contain "View list" link: https://github.com/civiform/civiform/issues/3033#issuecomment-1223383317 Do you think I need to clarify that? I don't have context on the original implementation of linkifying the count.

Reply:
Ah, those are likely way more up to date than my knowledge. Probably safe to go with what you have :D